setup.py

```diff
@@ -0,0 +1,22 @@
+from setuptools import setup
+
+import elife_api
+
+setup(
+    name='elife_api',
+    version=elife_api.__version__,
+    description='Utility for accessing eLife API-Raml JSON schemas in Python.',
```

Comment:
Maybe just `Utility for accessing eLife API JSON Schemas.`?

Reply:
Ok changed.